### PR TITLE
Use v1helper.GetPersistentVolumeClaimClass for compatibility

### PR DESCRIPTION
**Pull request by mrkm4ntr**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> 

/kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
To get class name of PVC, we should use beta annotation first until it has been obsoleted at v2.
https://github.com/kubernetes/kubernetes/pull/51440#issuecomment-396640304

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/controller/volume/scheduling/scheduler_binder.go [hunk 1]
```diff
@@ -665,11 +665,7 @@ func (b *volumeBinder) findMatchingVolumes(pod *v1.Pod, claimsToBind []*v1.Persi
 
 	for _, pvc := range claimsToBind {
 		// Get storage class name from each PVC
-		storageClassName := ""
-		storageClass := pvc.Spec.StorageClassName
-		if storageClass != nil {
-			storageClassName = *storageClass
-		}
+		storageClassName := v1helper.GetPersistentVolumeClaimClass(pvc)
```
Comment (msau42) [inline pkg/controller/volume/scheduling/scheduler_binder.go]: I think https://github.com/kubernetes/kubernetes/blob/c5635753ea5b148b6f416d5692beeb221f29a76e/pkg/scheduler/algorithm/predicates/csi_volume_predicate.go also has to be updated

Comment (mrkm4ntr) [reply pkg/controller/volume/scheduling/scheduler_binder.go]: Thank you for your review. I fixed it.

**Comment by mrkm4ntr** [comment]
@ravisantoshgudimetla Could you review this?

**Review by ravisantoshgudimetla — APPROVED** [review]
/lgtm
/approve

Thanks for the cleanup @mrkm4ntr

**Pull request merged**